<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClueUIClass</class>
 <widget class="QMainWindow" name="ClueUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClueUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="possessionTable">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>22</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>18</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>18</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>22</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="accusationLabel">
            <property name="text">
             <string>Accusation:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="undoHistoryLabel">
            <property name="text">
             <string>Undo history size: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="sleuthingLabel">
        <property name="text">
         <string>Sleuthing Tools</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="addTheoryButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add Theory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="undoButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="theoryLabel">
        <property name="text">
         <string>Possession Theories</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="theoryTable"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Game"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
